File: src/app/interfaces/theme/rules/Nord.java
package app.interfaces.theme.rules;

import app.interfaces.theme.Refresh;
import com.formdev.flatlaf.intellijthemes.FlatNordIJTheme;

import javax.swing.*;

public class Nord implements Refresh {

  @Override
  public void refresh(java.awt.Window frame) {
    try {
      UIManager.setLookAndFeel(new FlatNordIJTheme());
      SwingUtilities.updateComponentTreeUI(frame);
    } catch (javax.swing.UnsupportedLookAndFeelException | NullPointerException e ) {
      // do nothing
      e.addSuppressed(e);
    }
    frame.pack();
  }

  public String toString() {
    return "nord";
  }
}

File: kt/app/interfaces/theme/rules/Nord.kt
package app.interfaces.theme.rules

import app.interfaces.theme.Refresh
import com.formdev.flatlaf.intellijthemes.FlatNordIJTheme
import java.awt.Window
import javax.swing.SwingUtilities
import javax.swing.UIManager
import javax.swing.UnsupportedLookAndFeelException

class Nord : Refresh {
    @Throws(UnsupportedLookAndFeelException::class)
    override fun refresh(frame: Window) {
        UIManager.setLookAndFeel(FlatNordIJTheme())
        try {
            SwingUtilities.updateComponentTreeUI(frame)
        } catch (e: NullPointerException) {
            // do nothing
            e.addSuppressed(e)
        }
        frame.pack()
    }

    override fun toString(): String {
        return "nord"
    }
}